source/how-tos/change-boot-option.rst

```diff
@@ -3,29 +3,108 @@
+SBL has a default list of boot options. A example list is provided
+below. It attempts to boot from the first entry. If it fails to load
+anything bootable, it tries the subsequent options. If none of the boot
+options are working, it fallbacks to shell. When SBL is built for
+release mode, fallback to Shell is disabled for security considerations.
 
-#. eMMC boot partition 0
-#. eMMC boot partition 1
-#. SATA boot partition 0
-#. USB boot partition 0
+Example Boot Options List:
 
+1. eMMC boot partition 0
 
+2. eMMC boot partition 1
+
+3. SATA boot partition 0
+
+4. USB boot partition 0
+
+Boot options can be pre-configured during build or they can be
+configured post-build and during image stitch.
 
 Change at Pre Build Time
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Manually modify ``CfgData_BootOption.dsc`` and rebuild |SPN|.
+Manually modify CfgData\_BootOption.dsc and rebuild SBL.
+
+**Step1**. In the console launch the "ConfigEditor" from
+"slimbootloader" folder.
+
+**Command** : python Sblopen/BootloaderCorePkg/Tools/ConfigEditor.py
+
+.. image:: /images/Boot_options_1.jpg
+
+**Step2.** Open the "Config DSC" file, then select the description file
+according to your platform( eg.
+/Platform/CoffeelakeBoardPkg/CfgData/CfgDataDef.dsc")
+
+.. image:: /images/Boot_options_2.jpg
+
+
+.. image:: /images/Boot_options_3.jpg
+
+**Step 3.**  Load the "Config Delta" file from the same folder of the
+description file by selecting "Load Config Changes from Delta File"
+menu.
+
+.. image:: /images/Boot_options_4.jpg
+
+.. image:: /images/Boot_options_5.jpg
+
+**Step 4**. At the menu panel in left, select the "OS Boot options", and
+go the Boot option number which needs to be updated.
+
+.. image:: /images/Boot_options_6.jpg
+
+Example: Update boot device 0 to usb
+
+.. image:: /images/Boot_options_7.jpg
+
+**Step 5.** Save the changes with "**save config changes to Delta
+File**\ ” to the required board specific dlt file
+
+.. image:: /images/Boot_options_8.jpg
+
+.. image:: /images/Boot_options_9.jpg
+
+After saving, changes shall be reflected in the dlt file. Open the
+corresponding dlt file in notepad and check if the value is updated
+accordingly.
+
+On boot up,check the shell to see if the changes are properly reflected
+
 
 
 Change at Post Build time
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use |CFGTOOL| to modify a boot option entry then stitch 
-the update configuration data component in |SPN|.
+Use Configuration Editor to modify a boot option entry then stitch the
+update configuration data component in SBL.
+
+update IFWI with new Configuration
 
-See more details in :ref:`configuration-tool`.
+Follow steps 1 to 6 .Changes get saved into the dlt file .Copy the
+generated .dlt file into the same directory where CfgDataStitch.py is
+located
 
+Run CfgData Stitch Utility
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Config data Stitch utility is run using CfgDataStitch.py script::

```

Comment:
Need to tell where to get CfgDataStitch.py.  It needs to be the one generated in the FV folder.  Not the one in our Tools script folder.  This one in tool folder is a template.